sesman/libscp/libscp_session.c

```diff
@@ -416,6 +416,21 @@ scp_session_set_addr(struct SCP_SESSION *s, int type, const void *addr)
 }
 
 /*******************************************************************/
+int
+scp_session_set_guid(struct SCP_SESSION *s, const tui8 *guid)
+{
+    if (0 == guid)
```

Comment:
I know what you mean here, the NULL means it's a pointer.  Currently, zeros are used all over the project.  I've never liked that NULL is defined differently sometimes.